src/main/java/blackjack/service/BlackJackService.java

```diff
@@ -0,0 +1,51 @@
+package blackjack.service;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.Cards;
+import blackjack.domain.player.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class BlackJackService {
+
+    private Challengers challengers;
+    private Dealer dealer;
+
+    public void initChallengers(final List<String> requestNames) {
+        List<Challenger> challengers = new ArrayList<>();
+        for (String name : requestNames) {
+            challengers.add(new Challenger(getInitCards(), new Name(name)));
```

Comment:
```suggestion
            challengers.add(new Challenger(getInitCards(), name));
```

밖에서는 `Name`의 존재를 모를 수도 있지 않을까요? 이렇게 되면 어떤 장점이 있을까요?

Reply:
밖에서는 Name의 존재를 모를수도 있지 않는다는 말이 잘 와닿질 않는데, 설명해주실수 있을까요?
Name이라는 클래스를 따로 만드는 것에 대한 질문이라면,
Name클래스를 가짐으로서, 그 안에서 validation 및 빈칸에 대한 trimming을 하고 싶어서 따로 클래스 분리를 하였어요.

Reply:
제 코멘트는 `Name` 객체를 없애라는 이야기가 아니었어요. `Challenger` 객체를 만들 때, 생성자에 `Name` 객체를 직접 만들어 넣어주지 않고 `String`만 넣어줘도 되지 않을까 하는 제안이었답니다. 이 `String`을 받아 `Challenger` 생성자 내부에서 `Name`을 생성하는 것이죠. 이렇게 하면 `Challenger`를 사용하는 입장에서는 더 편리하지 않을까요? 굳이 `Name`의 존재를 알 필요도 없고요.